index.html

```diff
@@ -15,21 +16,34 @@
           <a href="#">My Logo </a>
         </div>
 
+          <ul class="nav-menu">
+
+            <li class="mobile-item">
+              <a href="#" class="mobile-list">Portfolio</a>
+            </li>
+            <li class="mobile-item">
+              <a href="#" class="mobile-list">About</a>
+            </li>
+            <li class="mobile-item">
+              <a href="#" class="mobile-list">Contact</a>
+            </li>
+            
+          </ul>
+        
```

Comment:
- [ ] Kindly make sure when the user clicks any of the mobile menu options, the correct part of the page is displayed as per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=When%20the%20user%20clicks%20(or%20taps)%20any%20of%20the%20mobile%20menu%20options%2C%20a%20correct%20part%20of%20the%20page%20is%20displayed.).